docs/rules-engine/examples/runtime-facts.md

```diff
@@ -1,92 +1,238 @@
-                'elementType': 'ACTION',
-                'actionType': 'SET_FACT',
-                'fact': 'UI_FACT_2',
-                'value': true
+                "actionType": "SET_FACT",
+                "key": "IS_MINOR",
+                "value": "TRUE"
               }
             ],
-            'failureElements': [
+            "failureElements": [
               {
-                'elementType': 'ACTION',
-                'actionType': 'SET_FACT',
-                'fact': 'UI_FACT_2',
-                'value': false
+                "actionType": "SET_FACT",
+                "key": "IS_MINOR",
+                "value": "FALSE"
               }
             ]
           }
         },
         {
-          'id': '6e8t54h6s4e-6erth46sre8th4-d46t8s13t5j2',
-          'name': 'the second rule',
-          'inputRuntimeFacts': ['UI_FACT_2'],
-          'inputFacts': [],
-          'outputRuntimeFacts': [],
-          'rootElement': {
-            'elementType': 'RULE_BLOCK',
-            'blockType': 'IF_ELSE',
-            'condition': {
-              'any': [
+          "id": "5467e501-b9ff-414f-8026-56885d0d7aas",
+          "name": "Teaser for minors in Paris and in New-York",
+          "disabled": false,
+          "outputRuntimeFacts": [],
+          "inputRuntimeFacts": [
+            "IS_MINOR"
+          ],
+          "rootElement": {
+            "elementType": "RULE_BLOCK",
+            "blockType": "IF_ELSE",
+            "condition": {
+              "all": [
                 {
-                  'lhs': {
-                    'type': 'RUNTIME_FACT',
-                    'value': 'UI_FACT_2'
+                  "lhs": {
+                    "type": "RUNTIME_FACT",
+                    "value": "IS_MINOR"
                   },
-                  'operator': 'equals',
-                  'rhs': {
-                    'type': 'LITERAL',
-                    'value': false
+                  "operator": "equals",
+                  "rhs": {
+                    "type": "LITERAL",
+                    "value": "TRUE"
                   }
-                }
-              ]
+                },
+                {
+                  "lhs": {
+                    "type": "FACT",
+                    "value": "locationOfConference"
+                  },
+                  "operator": "inArray",
+                  "rhs": {
+                    "type": "LITERAL",
+                    "value": ["Paris", "New-York"]
+                  }
+                },
+              ],
             },
-            'successElements': [
-              // Any action
-            ],
-            'failureElements': [
-              // Any action
+            "successElements": [
+              {
+                "actionType": "UPDATE_PLACEHOLDER",
+                "key": "teaser-placeholder",
+                "value": "assets/placeholders/teaser-activities.json"
+              }
             ]
           }
         }
       ]
     },
+    {
```

Comment:
I think this rule has to be in the ruleset above. As the indentation is done now, I think it is at the same level as the ruleset definition.